util/cron/common-python.bash

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# a helper function to select python versions before running `nightly` but
+# *after* sourcing other `common*.bash`
+function set_python_version() {
+  local ver_str=$1
```

Comment:
This is all good as-is, but I did wonder if it would be simpler to just pass in major/minor as separate args, instead of parsing a string. (Probably not worth changing, since this has been tested)

Reply:
Yeah, there wasn't anything smart behind using string arguments. And in hindsight, what you are suggesting might have been simpler.